scripts/eslint-templates/src.eslintrc

```diff
@@ -29,6 +27,12 @@ rules:
   - error
   - code: 140
 
+  nonblock-statement-body-position:
+  - error
+  - below
+  implicit-arrow-linebreak:
```

Comment:
`implicit-arrow-linebreak` disabled because code didn't look too good with it enabled